src/components/forms/sections/WinterBoatDetails.tsx

```diff
@@ -10,15 +10,39 @@ import WinterUnRegisteredBoat from '../tabs/WinterUnRegisteredBoat';
 import { WinterStorageMethod } from '../../../__generated__/globalTypes';
 import { WinterFormValues } from '../../../types/winterStorage';
 import { WithBoatType } from '../Selects';
-import SectionSelector from './SectionSelector';
+import SectionSelector, { TypeProps } from './SectionSelector';
```

Comment:
It seems that this paragraph is missing:
<img width="647" alt="Screenshot 2020-08-31 at 14 22 12" src="https://user-images.githubusercontent.com/1407958/91714997-87401600-eb95-11ea-886f-0be4b648eb07.png">

Reply:
Added, also to the list of texts to be translated